src/x11/ewmh.cpp

```diff
@@ -77,6 +77,13 @@ namespace ewmh_util {
     return desktop;
   }
 
+  unsigned int get_number_of_desktops(int screen) {
```

Comment:
Just to make it consistent with what `xcb_ewmh.h` uses.

```suggestion
  uint32_t get_number_of_desktops(int screen) {
```